canonical-web-and-design/vanilla-framework
# Add individual examples for layout patterns

## Done

Makes sure "layout" related patterns can be built separately and add individual examples for them.

Fixes #2751

## QA

NOTE: Visual side of the examples listed below was verified by CI on Percy, so QA steps are to make sure examples work as expected and detailed visual review of those is not needed.

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/ or [demo](https://vanilla-framework-canonical-web-and-design-pr-2763.run.demo.haus/)
- Go to [examples page](https://vanilla-framework-canonical-web-and-design-pr-2763.run.demo.haus/)
- Make sure that existing examples (full Vanilla) for components below work as before (check different variants for each pattern):
cards,
strips,
matrix,
modal
- Make sure that individual examples for these components work the same as full ones
